src/foam/u2/Element.js

```diff
@@ -2322,6 +2322,10 @@ foam.CLASS({
         The order to render the property in if rendering multiple properties.
       `,
       value: Number.MAX_SAFE_INTEGER
+    },
+    {
+      class: 'Boolean',
+      name: 'visibilityPermissionRequired'
```

Comment:
How is this different than readPermissionRequired?

Reply:
readPermissionRequired check if a permission is required for user to access this property
visibilityPermissionRequired check if user need permission to see it in GUI, like a column in table view